### PR TITLE
Add a stage for linting .travis.yml

### PR DESCRIPTION
**Pull request by prymitive**
We shouldn't run CI job if CI job config is borked, this should help catch that

### CODE REVIEW

**Comment by terinjokes** [comment]
I think that if the YAML is borked, this would never run anyways.

**Comment by prymitive** [comment]
I'm hoping it would catch issues like missing required keys rather than YAML syntax

File: .travis.yml [hunk 1]
```diff
@@ -2,6 +2,12 @@ go_import_path: github.com/cloudflare/unsee
 
 jobs:
   include:
+    - stage: Lint Travis CI configuration
+      before_script:
+        - gem install travis --no-rdoc --no-ri
+      script:
+        - travis lint --no-interactive .travis.yml
```
Comment (terinjokes) [inline .travis.yml]: I think you'll need to add `-x` to have it set the exit code.

**Comment by prymitive** [comment]
Looks like it doesn't work with stages yet - https://github.com/travis-ci/travis.rb/issues/571

**Comment by prymitive** [comment]
Stages feature needs to go out of beta first, closing for now

**Pull request closed without merging**